modules/appnexusBidAdapter.js

```diff
@@ -316,6 +327,22 @@ function newBid(serverBid, rtbBid, bidderRequest) {
       vastImpUrl: rtbBid.notify_url,
       ttl: 3600
     });
+
+    const videoContext = utils.deepAccess(
+      bidderRequest.bids[0],
```

Comment:
We can't always assume the first bid object will be an `adpod`; a publisher could be mixing the ad-types together (like `banner` + `adpod`) in the adunit.  

The local `bidRequest` variable ([here](https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L294)) looks to already pull the matching bid based on the id (if I'm reading that right); could you take a look at this var and see if it meets the needs here?